torch/quantization/quantize.py

```diff
@@ -333,3 +333,26 @@ def swap_module(mod, mapping):
         if type(mod) in mapping:
             new_mod = mapping[type(mod)].from_float(mod)
     return new_mod
+
+def dump_tensor(mod, target_dict, prefix=""):
+    r"""Traverse the modules and save the weight and stored activation to given dict.
+    This is mainly used for quantization accuracy debug
+    Args:
+        mod: the top module we want to save all tensors
+        prefix: the prefix for the current module
+        target_dict: the dictionary used to save the tensors
+    """
+    def get_prefix(prefix):
+        return prefix if prefix == "" else prefix + '.'
+
+    weight_unpack = getattr(mod, "weight", None)
```

Comment:
why do we need to handle the weights part here and hard-code the names of parameters? module.state_dict() should give you what you need for this debug